### PR TITLE
[#147] image improvements

### PR DESCRIPTION
**Pull request by rbeer**
### Issue
#147 - image improvement

- [x]  Move content all images from the src/ folder to the respective doc folder
55dde14

- [x]  ~Show how an image would look resized using parameters in the URL~
4ebe583

- [x]  Document the custom syntax options added here in the repo README0.001
241d633, 9a2b129, b6a96c3

### Solution
Using plugins for `remark` and `Gatsby` to parse custom Markdown syntax and transform AST 'image' nodes to HTML.

[remark-image-attributes](https://github.com/rbeer/remark-image-attributes) parses Markdown for image definitions,
```md
![satisfied](https://foomoji.com/satisfied.png#width=32px;height=32px)
```
producing
```js
{
  type: 'image',
  alt: 'satisfied',
  title: 'satisfied',
  url: 'https://foomoji.com/satisfied.png',
  attributes: { width: '32px', height: '32px' }
},
```

[gatsby-remark-image-attributes](https://github.com/rbeer/gatsby-remark-image-attributes) reads those AST 'image' nodes and transforms them to usable HTML, applying [styleAttributes](https://github.com/rbeer/gatsby-remark-image-attributes#styleattributes)
```html
<img
  src="https://foomoji.com/satisfied.png"
  alt="satisfied"
  style="width: 32px; height:32px;"
/>
```
This includes images already processed by e.g. [gatsby-remark-images](https://github.com/gatsbyjs/gatsby/master/packages/gatsby-remark-images/), which itself produces HTML and deletes the 'image' node.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,21 @@ To become a part of the Centrifuge community, read the [Code of Conduct](https:/
 
 ### Images
 
-Place all the static images under `src/images` folder and
+Place all the static images under `docs/images` folder and
```
Comment (lucasvo) [inline README.md:33]:
```suggestion
Place all the static images under `docs/` folder close to the relevant content and
```

**Comment by rbeer** [comment]
Something broke; nothing is working anymore.
I'll have to debug this later or tomorrow; sorry.

**Comment by rbeer** [comment]
So this works for me locally, but it's like the new plugins aren't present in the [deploy preview](https://deploy-preview-172--staging-centrifuge-documentation.netlify.com/).

**Comment by rbeer** [comment]
9d4fa42 works; I'll publish the new version and rebase

**Pull request merged**